src/styles/projects.module.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Lilita+One|Quicksand:300&display=swap');
+
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.innerContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 4% 0;
+}
+
+.font {
+    font-family: 'Quicksand', sans-serif;
+    align-items: center;
+}
+
+.projectContainer {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    height: 350px;
+    width: 100%;
+}
+
+.detail {
+    width: 35%;
+    /* background-color: blue; */
+    height: 350px;
+}
+
+.font {
+    font-family: 'Quicksand', sans-serif;
+}
+
+.link {
+    padding: 2px;
+    /* border: 2px solid lightgrey; */
+    border-radius: 25px;
+    cursor: pointer;
+    display: block;
+    margin: 5px;
+}
+
+a {
```

Comment:
Using a selector like `a` is vulnerable in general, as you can have so many `a` tag in your component that you want to apply different style. Or someone else may add `a` tags which will include your style unexpectedly.

Defining something like `a` should be in global if you want to have a default style for it.